api.d.ts

```diff
@@ -186,6 +186,53 @@ export interface BundlingAPI {
 
 export declare const bundling: BundlingAPI
 
+// ----------------------------------------------------------------
+// -------------------------- SDK ----------------------------
+// ----------------------------------------------------------------
+
+export interface SdkAPI {
+  /**
+   * create SDK json files
+   */
+  generateSdkDocumentation: (
+    target: string,
+    options?: {
+      /**
+       * Output directory for the sdk.
+       * Default value is "dist".
+       */
+      outDir?: string
+
+      /**
+       * Source files to create the sdk from.
```

Comment:
The comment here talks about inclusion but the config key is about exclusion.
What are we excluding and why? what use case does this config option serve?